plugins/woocommerce/src/Blocks/Shipping/ShippingController.php

```diff
@@ -433,6 +430,42 @@ function ( $rate ) {
 		return $packages;
 	}
 
+	/**
+	 * Remove shipping if no address is entered.
```

Comment:
Even though we just paired on this, it took me a minute to remember what this was for and figure out why. I think the comment could be updated, I made a suggestion:

```suggestion
	 * Remove shipping (i.e. delivery, not local pickup) if "Hide shipping costs until an address is entered" is enabled, and no address has been entered yet.
```